### ISSUE TITLE
Compile failure in midi.rs

### ISSUE THREAD

**Issue by rdaum** [RustAudio/rimd#7]
Using Rust toolchain version 1.23.0 on Windows,  the following in my Cargo.toml:

```
[dependencies.rimd]
git = "https://github.com/RustAudio/rimd.git"
```

And use of rimd:

`extern crate rimd;`

```
error[E0599]: no function or associated item named `from_u8` found for type `midi::Status` in the current scope
   --> C:\Users\ryand\.cargo\git\checkouts\rimd-f9f9642f9e33bd77\9ecb792\src\midi.rs:103:9
    |
103 |         Status::from_u8(self.data[0] & STATUS_MASK).unwrap()
    |         ^^^^^^^^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:
            candidate #1: `use enum_primitive::FromPrimitive;`

error[E0599]: no function or associated item named `from_u8` found for type `midi::Status` in the current scope
   --> C:\Users\ryand\.cargo\git\checkouts\rimd-f9f9642f9e33bd77\9ecb792\src\midi.rs:158:15
    |
158 |         match Status::from_u8(status & STATUS_MASK) {
    |               ^^^^^^^^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:
            candidate #1: `use enum_primitive::FromPrimitive;`

error[E0599]: no function or associated item named `from_u8` found for type `meta::MetaCommand` in the current scope
   --> C:\Users\ryand\.cargo\git\checkouts\rimd-f9f9642f9e33bd77\9ecb792\src\meta.rs:158:19
    |
158 |             match MetaCommand::from_u8(try!(read_byte(reader))) {
    |                   ^^^^^^^^^^^^^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:
            candidate #1: `use enum_primitive::FromPrimitive;`

error: aborting due to 3 previous errors

error: Could not compile `rimd`.
```

**Comment by Boscop** [comment]
Probably crate version mismatch. Does it work with this one? (It works for me) https://github.com/RustAudio/rimd/pull/8

**Comment by mitchmindtree** [comment]
@rdaum closing this assuming that this is fixed by #8. Feel free to re-open if you're still running into the issue!
